bloom-housing/bloom
# feat: exposes pref unit type

This PR addresses https://github.com/metrotranscom/doorway/issues/894


## Description
exposes pref unit types for application list

## How Can This Be Tested/Reviewed?
log into the partner site, open your browser's console -> network tab 
head into a listing -> application tab -> verify that if an application had pref unit type filled out that data was returned

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
